bundle/jsky.app.ot/src/main/java/jsky/app/ot/gemini/editor/targetComponent/EdCompTargetList.java

```diff
@@ -821,7 +823,11 @@ private void setSelectionFromNode() {
             if (selectionIsBasePosition()) {
                 DialogUtil.error("You can't remove the base position.");
             } else if (selectionIsAutoTarget()) {
-                DialogUtil.error("You can't remove automatic guide stars.");
+                AgsStrategyUtil.setSelection(
+                        getContextObservation(),
+                        ImOption.fromScalaOpt(AgsRegistrar.lookup(AgsStrategyKey$.MODULE$.fromStringOrNull("Off")))
```

Comment:
Could you just pass in the `AgsStrategKey.OffKey` directly or is it a problem accessing that in Java.

The `Off` strategy is just stubbed out so it seems like it could easily go in the `AgsStrategy` companion object or even in the `edu.gemini.ags.api` package as a top-level class and then you wouldn't need to look it up at all.